dice.js

```diff
@@ -0,0 +1,15 @@
+function rollDice(rolls){
+    let sumRandoms = 0
+    let randomNum = []
+    for (let i = 0; i < rolls; i++){
+     randomNum.push(Math.round(Math.random() * 6) + 1)
+}
+
+for (let x = 0; x < randomNum.length; x++){
```

Comment:
Great solution Shahad, Also there another way to iterate through the array to get the total of numbers is called ```reduce``` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce